src/catalog_page/content.js

```diff
@@ -1,38 +1,50 @@
 import React from 'react';
 
 import './content.css';
-import logo0 from '../img/logo.png'
-import logo1 from '../img/logo1.jpg'
-import logo2 from '../img/logo2.jpg'
-import logo3 from '../img/logo3.jpg'
-import logo4 from '../img/logo4.jpg'
-import logo5 from '../img/logo5.jpg'
+
+
+import logo0 from '../img/7.jpg'
```

Comment:
Suggestion:
create index.js inside of img folder, that will reexport images so you can import them more easily inside of components.